packages/sku/lib/program/commands/init/init.command.js

```diff
@@ -0,0 +1,14 @@
+const { Command } = require('commander');
+
+const init = new Command('init');
+
+init
+  .description('Initialize a new sku project')
+  .argument('[packageName]', 'Package name')
+  .option('--verbose', 'Verbose log output')
+  .action((packageName, verbose) => {
+    const initAction = require('./init.action');
+    initAction(packageName, { verbose });
```

Comment:
Do async actions need to be awaited, or does `commander` handle this for you/not care?

Reply:
We don't care if we handle errors internally. If we want to add some custom error handling in commander we may have to switch to using async functions.

Reply:
> I'm more concerned with whether commander will wait for the action to finish

What should it do once the action is finished? From my understanding is that it does nothing when the action finishes. If the action throws it does something but in this case we're not awaiting the action so any error thrown inside will not be captured by commander.

We don't need to await the `parseAsync` I think. We only want to await it if we're doing something else after the program has finished.

I.E.

```js
// sync.
program.parse();
// program finished here. Your actions are done.
console.log('do something after');

// async.
program.parseAsync();
// program not finished here. Console log firing before your program has finished running.
console.log('do something after');
```

I'll have to test it to see if this is the case. This is how I understand it.